src/Klarna/Rest/MerchantCardService/VCCSettlements.php

```diff
@@ -108,7 +109,9 @@ public function retrieveSettlement($settlementId, $keyId)
             'GET',
```

Comment:
Use constant of Method class instead of string

Reply:
Replaced all the text methods in all test files.